test/fixtures/users.yml

```
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+foobar:
+  firstname: "Foo"
+  lastname: "Bar"
+  email: "foo@bar.com"
+  room: "A200"
```

I just had an idea about our fixtures: we could use fictional characters, that way we have something more memorable and easier to think about than "Foo Bar", but still without any personal information. Example:
```suggestion
ironman:
  firstname: "Tony"
  lastname: "Stark"
  email: "tony@avengers.com"
  room: "A200"
```

---

I like this idea but i hope you are not suggesting this because you are sponsored by Disney